frontend-react/README.md

```diff
@@ -109,6 +111,33 @@ NOTE: This only works `run:build-dir` because webpack's dynamic runtime updating
 
 `.github/workflows/chromatic-pr.yml` triggers a Chromatic build anytime a file with `// AutoUpdateFileChromatic` comment on its FIRST LINE is checked in to a PR. The goal here is to automatically update our Chromatic anytime a file that has an associated Storybook is modified.
 
+## Run the e2e tests
+
+[Playwright](https://playwright.dev/) is the framework used to create and run our e2e tests.
+
```

Comment:
nit: might be worth just adding the line that you have to explicitly install Playwright to get these tests working + that the test code hot updates.